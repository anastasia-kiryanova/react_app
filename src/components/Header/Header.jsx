import React from 'react'
import { AppBar, Toolbar, Typography } from '@mui/material/'

export const Header = () => {
  return (
    <AppBar position="sticky" color="primary">
      <Toolbar>
        <Typography variant="h5" component="div">
          Добро пожаловать в чат
        </Typography>
      </Toolbar>
    </AppBar>
  )
}