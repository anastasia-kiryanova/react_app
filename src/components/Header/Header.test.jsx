import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react';
import { Header } from './Header';

describe('Header', () => {
  it('is a Component', () => {
    expect(Header).toBeInstanceOf(Function);
  });

  it ('test snapshot', () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  });

  it('renders Header component', () => {
    render(<Header />);
    expect(screen.getByTestId('header-text')).toBeTruthy();
  });
});