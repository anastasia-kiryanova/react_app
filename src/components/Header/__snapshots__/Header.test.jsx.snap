// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header test snapshot 1`] = `
<DocumentFragment>
  <header
    class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionSticky css-7e0s21-MuiPaper-root-MuiAppBar-root"
  >
    <div
      class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-1t29gy6-MuiToolbar-root"
    >
      <div
        class="MuiTypography-root MuiTypography-h5 css-ag7rrr-MuiTypography-root"
        data-testid="header-text"
      />
    </div>
  </header>
</DocumentFragment>
`;
